simulation/g4main/HepMCNodeReader.cc

```diff
@@ -144,178 +148,284 @@ int HepMCNodeReader::process_event(PHCompositeNode *topNode)
+		  cout << "z: " << zpos << endl;
+		  cout << "t: " << time << endl;
+		  cout << "Particles" << endl;
+		}
 
         if (_position_filter_on &&
             (xpos < _pos_filter_x_min || xpos > _pos_filter_x_max ||
              ypos < _pos_filter_y_min || ypos > _pos_filter_y_max ||
              zpos < _pos_filter_z_min || zpos > _pos_filter_z_max   )) continue;
         
-        if (ishape == ShapeG4Tubs)
-        {
-          if (sqrt(xpos * xpos + ypos * ypos) > worldsizey / 2 ||
-              fabs(zpos) > worldsizez / 2)
-          {
-            cout << "vertex x/y/z" << xpos << "/" << ypos << "/" << zpos
-                 << " outside world volume radius/z (+-) " << worldsizex / 2
-                 << "/" << worldsizez / 2 << ", dropping it and its particles"
-                 << endl;
-            continue;
-          }
-        }
-        else if (ishape == ShapeG4Box)
-        {
-          if (fabs(xpos) > worldsizex / 2 || fabs(ypos) > worldsizey / 2 ||
-              fabs(zpos) > worldsizez / 2)
-          {
-            cout << "Vertex x/y/z " << xpos << "/" << ypos << "/" << zpos
-                 << " outside world volume x/y/z (+-) " << worldsizex / 2 << "/"
-                 << worldsizey / 2 << "/" << worldsizez / 2
-                 << ", dropping it and its particles" << endl;
-            continue;
-          }
-        }
-        else
-        {
-          cout << PHWHERE << " shape " << ishape << " not implemented. exiting"
-               << endl;
-          exit(1);
-        }
-
-        // For pile-up simulation: vertex position
-        vtxindex = ineve->AddVtx(xpos, ypos, zpos, time);
-        int trackid = -1;
-        for (fiter = finalstateparticles.begin();
-             fiter != finalstateparticles.end();
-             ++fiter)
-        {
-          ++trackid;
-
-          if (verbosity > 1) (*fiter)->print();
-
-          PHG4Particle *particle = new PHG4Particlev1();
-          particle->set_pid((*fiter)->pdg_id());
-          particle->set_px((*fiter)->momentum().px() * mom_factor);
-          particle->set_py((*fiter)->momentum().py() * mom_factor);
-          particle->set_pz((*fiter)->momentum().pz() * mom_factor);
-          particle->set_barcode((*fiter)->barcode());
-
-          ineve->AddParticle(vtxindex, particle);
-
-          if (embed_flag != 0) ineve->AddEmbeddedParticle(particle, embed_flag);
-        }
-      }  //      if (!finalstateparticles.empty())
-
-    }  //    for (HepMC::GenEvent::vertex_iterator v = evt->vertices_begin();
-
-  }  // For pile-up simulation: loop end for PHHepMC event map
+	      if (ishape == ShapeG4Tubs)
+		{
+		  if (sqrt(xpos * xpos + ypos * ypos) > worldsizey / 2 ||
+		      fabs(zpos) > worldsizez / 2)
+		    {
+		      if (Verbosity()>0)
+		      cout << "vertex x/y/z" << xpos << "/" << ypos << "/" << zpos
+			   << " outside world volume radius/z (+-) " << worldsizex / 2
+			   << "/" << worldsizez / 2 << ", dropping it and its particles"
+			   << endl;
```

Comment:
Please make the line indent right.  I'd suggest to also use `{}` for multiple lines (even in case of a single command).